spark/src/main/scala/org/apache/spark/sql/delta/actions/actions.scala

```diff
@@ -1193,7 +1204,8 @@ case class CommitInfo(
     userMetadata: Option[String],
     tags: Option[Map[String, String]],
     engineInfo: Option[String],
-    txnId: Option[String]) extends Action with CommitMarker with AbstractCommitInfo {
+    txnId: Option[String])
+  extends Action with CommitMarker with SparkAbstractCommitInfo with StorageAbstractCommitInfo {
```

Comment:
I am confused. Can you explain more on the intention?
The CommitInfo here extends a java interface
```
public interface AbstractCommitInfo {
  long getCommitTimestamp();
}
```
and a scala interface
```
trait AbstractCommitInfo {
  def getCommitTimestamp: Long
}
```